Commands/InteractionCommands/JoinwatchInteractions.cs

```diff
@@ -8,85 +8,24 @@ public class JoinwatchSlashCmds
         {
             [SlashCommand("add", "Watch for joins and leaves of a given user. Output goes to #investigations.")]
             public async Task JoinwatchAdd(InteractionContext ctx,
-                [Option("user", "The user to watch for joins and leaves of.")] DiscordUser user,
-                [Option("note", "An optional note for context.")] string note = "")
+                [Option("user", "The user to watch for joins and leaves of.")] DiscordUser _,
+                [Option("note", "An optional note for context.")] string __ = "")
             {
-                var joinWatchlist = await Program.db.ListRangeAsync("joinWatchedUsers");
-
-                if (joinWatchlist.Contains(user.Id))
-                {
-                    // User is already watched
-
-                    // Get current note; if it's the same, do nothing
-                    var currentNote = await Program.db.HashGetAsync("joinWatchedUsersNotes", user.Id);
-                    if (currentNote == note || (string.IsNullOrWhiteSpace(currentNote) && string.IsNullOrWhiteSpace(note)))
-                    {
-                        await ctx.RespondAsync($"{Program.cfgjson.Emoji.Error} {user.Mention} is already being watched with the same note! Nothing to do.");
-                        return;
-                    }
-
-                    // If note is different, update it
-
-                    // If new note is empty, remove instead of changing to empty string!
-                    if (string.IsNullOrWhiteSpace(note))
-                    {
-                        await Program.db.HashDeleteAsync("joinWatchedUsersNotes", user.Id);
-                        await ctx.RespondAsync($"{Program.cfgjson.Emoji.Success} Successfully removed the note for {user.Mention}! They are still being watched.");
-                    }
-                    else
-                    {
-                        await Program.db.HashSetAsync("joinWatchedUsersNotes", user.Id, note);
-                        await ctx.RespondAsync($"{Program.cfgjson.Emoji.Success} Successfully updated the note for {user.Mention}:\n> {note}");
-                    }
-                }
-                else
-                {
-                    // User is not joinwatched, watch
-                    await Program.db.ListRightPushAsync("joinWatchedUsers", user.Id);
-                    if (note != "")
-                        await Program.db.HashSetAsync("joinWatchedUsersNotes", user.Id, note);
-                    await ctx.RespondAsync($"{Program.cfgjson.Emoji.Success} Now watching for joins/leaves of {user.Mention} to send to the investigations channel"
-                        + (note == "" ? "!" : $" with the following note:\n>>> {note}"));
-                }
+                await ctx.RespondAsync($"{Program.cfgjson.Emoji.Error} This command is deprecated and no longer works; all joinwatches have been converted to notes. Please use `/note add` instead, with the `show_on_join_and_leave` option.");
```

Comment:
If we feel we have to accept arguments for this stub, could we spit them back to the user in the format of an example /note command?